region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -1091,7 +1089,6 @@
       "link": [4, 1],
       "name": "X-Ray Access SpikeJumps with HiJump",
       "requires": [
-        {"notable": "X-Ray Access SpikeJumps with HiJump"},
         "HiJump",
         "canTrickyJump",
```

Comment:
removes this from being notable, which moves the spike variant from expert to VH

Reply:
The problem with this in VH is that no one other than me does this strat, so they would expect logic to want them to damage boost over.  
The practical difference between the two is that you can retry the hijump jump once you make it this deep into the room, but the damage boost strat you cannot.  And dboosting to this point and not to the door doesn't look modelled, so you have extra health if you do that.